libraries/ms-drivers/src/ltc_afe_impl.c

```diff
@@ -104,6 +103,42 @@ static StatusCode prv_trigger_aux_adc_conversion(LtcAfeStorage *afe) {
   return spi_exchange(settings->spi_port, cmd, 4, NULL, 0);
 }
 
+static StatusCode prv_write_comm_register(LtcAfeStorage *afe, uint8_t device_cell) {
+  if (device_cell > ADS1259_NUM_PINS) {
+    return STATUS_CODE_OUT_OF_RANGE;
+  }
+  LtcAfeSettings *settings = &afe->settings;
+  LtcAfeWriteCommRegPacket packet = { 0 };
+  // Build WRCOMM Command
+  prv_build_cmd(LTC6811_WRCOMM_RESERVED, packet.wrcomm, SIZEOF_ARRAY(packet.wrcomm));
+  // Write 3 bytes of data to the COMM registers
+  // We send the same data 3 times since using the WRCOMM command forces us to send 3 bytes
+  // However, the mux only needs a byte of data
+  packet.reg.icom0 = LTC6811_ICOM_CSBM_LOW;
+  packet.reg.d0 = device_cell - 1;
+  packet.reg.fcom0 = LTC6811_FCOM_CSBM_LOW;
+
+  packet.reg.icom1 = LTC6811_ICOM_CSBM_LOW;
+  packet.reg.d1 = device_cell - 1;
+  packet.reg.fcom1 = LTC6811_FCOM_CSBM_LOW;
+
+  packet.reg.icom2 = LTC6811_ICOM_CSBM_LOW;
+  packet.reg.d2 = device_cell - 1;
+  packet.reg.fcom2 = LTC6811_FCOM_CSBM_HIGH;
+  prv_wakeup_idle(afe);
+  return spi_exchange(settings->spi_port, (uint8_t *)&packet, 4, NULL, 0);
+}
+
+static StatusCode prv_mux_enable_spi(LtcAfeStorage *afe) {
+  LtcAfeSettings *settings = &afe->settings;
+  // Setting sending STCOMM command
+  uint16_t stcomm = LTC6811_STCOMM_RESERVED;
+  uint8_t cmd[4] = { 0 };
+  prv_build_cmd(stcomm, cmd, SIZEOF_ARRAY(cmd));
+  prv_wakeup_idle(afe);
+  return spi_exchange(settings->spi_port, cmd, 4, NULL, 0);
```

Comment:
So we def need to do the write 72 clock cycles thing now that I'm looking at it again. Not sure if we can just make sure to not use spi for that time or if there's something we need to do to get that to happen ...